experiment/test_scheduler.py

```diff
@@ -201,7 +171,9 @@ def _test_create_trial_instance(benchmark, expected_image, expected_target,
         startup_script=expected_startup_script_path)
 
     with open(expected_startup_script_path) as file_handle:
-        assert file_handle.read() == expected_format_string.format(
+        content = file_handle.read()
+        script_for_docker = content[content.find('## Start docker.'):]
```

Comment:
maybe assert '## Start docker.' in content first.

Reply:
Good idea! done